src/lib/org_utils.js

```diff
@@ -212,13 +212,12 @@ const tablePartContainsCellId = (tablePart, cellId) =>
 const doesAttributedStringContainTableCellId = (parts, cellId) =>
   parts
     .filter(part => ['table', 'list'].includes(part.get('type')))
-    .some(
```

Comment:
This file contains only whitespace change from prettier and can be ignored during the review. Sorry for that.